scripts/perftune.py

```diff
@@ -694,10 +718,15 @@ def __learn_irqs_one(self, iface):
         """
         # filter 'all_irqs' to only reference valid keys from 'irqs2procline' and avoid an IndexError on the 'irqs' search below
         all_irqs = set(learn_all_irqs_one("/sys/class/net/{}/device".format(iface), self.__irqs2procline, iface)).intersection(self.__irqs2procline.keys())
-        fp_irqs_re = re.compile("\-TxRx\-|\-fp\-|\-Tx\-Rx\-|mlx\d+\-\d+@")
+        fp_irqs_re = re.compile("\-TxRx\-|\-fp\-|\-Tx\-Rx\-|mlx4-\d+@|mlx5_comp\d+@")
         irqs = list(filter(lambda irq : fp_irqs_re.search(self.__irqs2procline[irq]), all_irqs))
         if irqs:
-            irqs.sort(key=self.__intel_irq_to_queue_idx)
+            driver_name = self.__get_driver_name(iface)
+            perftune_print("Sorting IRQs by driver: {}".format(driver_name))
```

Comment:
This printout may be confusing when the NIC vendor is neither Mellanox nor Intel (e.g. Broadcom).
In that case you'll not be sorting by the `bnx2x` driver like the printout would suggest.

Let's change the printout to be "Driver X has been detected" and that's it. No need to say anything about the sorting.

Reply:
fixed in original commit, added " set proper message for driver detection "